pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -243,7 +243,7 @@ func (c *SyncController) getExistingLimitRange(ns *corev1.Namespace) (*corev1.Li
 	return limitRanger[0], nil
 }
 
-func (c *SyncController) getNamespaceResourceQuota(ns *corev1.Namespace) (v32.ResourceQuotaLimit, error) {
+func (c *SyncController) getNamespaceResourceQuotaLimit(ns *corev1.Namespace) (v32.ResourceQuotaLimit, error) {
```

Comment:
I think @MbolotSuse's point is that this went from getting a [NamespaceResourceQuota](https://github.com/rancher/rancher/blob/cf4364d2a5b09a9075910a3f3179ed178d3cde61/pkg/apis/management.cattle.io/v3/resource_quota_types.go#L8) to a [ResourceQuotaLimit](https://github.com/rancher/rancher/blob/cf4364d2a5b09a9075910a3f3179ed178d3cde61/pkg/apis/management.cattle.io/v3/resource_quota_types.go#L12) so the name should actually be `getResourceQuotaLimit`. There is already a function called `getNamespaceResourceQuotaLimit` which is called on the line below so I don't think this will compile.